### PR TITLE
Fix Placeholders

### PR DESCRIPTION
**Pull request by scouillard**
## Description

Fix Recordings, ManageUsers, ManageServerRooms, RoomConfiguration and Roles placeholders.

Also in this PR:
- Create a `RoundPlaceHolder` template for which you can enter the radius in pixel

TODO next PR: 
Replace the manually specified width/radius props in `Avatar` and `RoundPlaceholder` for a CSS class.

### CODE REVIEW

File: app/assets/stylesheets/application.bootstrap.scss [hunk 1]
```diff
@@ -582,12 +582,14 @@ input[type="range"]:focus::-moz-range-thumb {
...
 }
+
 .placeholder {
   border-radius: 2px;
 }
+
 .placeholder-xlg {
   border-radius: 3px;
-  min-height: 2.5em;
+  min-height: 2em;
```
Comment (farhatahmad) [inline app/assets/stylesheets/application.bootstrap.scss:592]: This messes up the spacing for the room name placeholder

File: app/javascript/components/shared_components/utilities/RoundPlaceholder.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Placeholder } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default function RoundPlaceholder({ radius, className }) {
+  return (
+    <Placeholder animation="glow">
+      <Placeholder style={{ height: radius, width: radius, borderRadius: '50%' }} className={className} bg="secondary" />
+    </Placeholder>
+  );
```
Comment (farhatahmad) [inline app/javascript/components/shared_components/utilities/RoundPlaceholder.jsx:10]: Creating components with manually specified heights and widths passed in as a prop is bad practice. We should be matching the style used in other components (logo and placeholder)

**Comment by farhatahmad** [comment]
I'm playing around with this locally and honestly, I'm not a huge fan. It results in essentially large blocks of placeholders appearing on your screen.

I feel like the role of a placeholder is to each in the transition and I don't feel like it's doing that here

File: app/javascript/components/shared_components/utilities/RoundPlaceholder.jsx (hunk 2 above)
Comment (scouillard) [reply app/javascript/components/shared_components/utilities/RoundPlaceholder.jsx:10]: Agreed. We could replace the width/radius by a fixed class. It needs to be done for Avatars as well.

**Comment by scouillard** [comment]
> I'm playing around with this locally and honestly, I'm not a huge fan. It results in essentially large blocks of placeholders appearing on your screen.
> 
> I feel like the role of a placeholder is to each in the transition and I don't feel like it's doing that here

Okay I'll see what I can do.

I agree - it could use some more details.

**Comment by scouillard** [comment]
Removed the full length placeholders.

**Comment by farhatahmad** [comment]
Damn - this is much better in my opinion. Great work!

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**